### DISCUSSION TITLE
Manage recurring event for a longer range

### DISCUSSION THREAD

**Discussion by bastienho** [bitfireAT/icsx5#434] [Ideas]
Hi

I figured that recurring events have a limited date range. After some (how many?) days, new occurrences are not synced.

Could you, at least one of these:

1. Increase the default range
2. add a general setting to set the sync range
3. add a per calendar setting to set the sync range
 
thank you!

**Comment by sunkup** [comment]
> I figured that recurring events have a limited date range

Not necissarily. If an event is infinitely recurring then it is saved as such (with the infinite recurrence rule) in android too and not expanded until a certain time frame into the future. It might just be your calendar app which does not display the recurring event far enough into the future. You could check whether there is a setting to show more days in the future.

Otherwise in order to help you out further it would be interesting for us to see the ics file you are synchronizing. Maybe you can provide it here?
